.github/actions/paths-filter/action.yml

```diff
@@ -29,6 +32,14 @@ outputs:
         steps.filter-common.outputs.github-actions-change == 'true' ||
         steps.filter-identity.outputs.identity-frontend-change == 'true'
       }}
+  operate-backend-changes:
+    description: Output whether Operate backend tests should be run based on GitHub event and files changed
+    value: >-
+      ${{
+        github.event_name == 'push' ||
+        steps.filter-common.outputs.github-actions-change == 'true' ||
```

Comment:
Sorry for not spotting earlier: We should also check for `steps.filter-common.outputs.maven-change == 'true'` since this output is meant to influence (java) backend builds using Maven